nix/modules/fedimintd.nix

```diff
@@ -133,14 +156,24 @@ let
 
       dataDir = mkOption {
         type = types.str;
-        default = "/var/lib/fedimintd/";
+        default = "/var/lib/fedimintd-${name}/";
         readOnly = true;
         description = lib.mdDoc ''
           Path to the data dir fedimintd will use to store its data.
           Note that due to using the DynamicUser feature of systemd, this value should not be changed
           and is set to be read only.
         '';
       };
+
+      nginx = {
+        enable = mkEnableOption "fedimint";
+        config = mkOption {
+          # TODO: change to something like https://github.com/NixOS/nixpkgs/pull/314440/files#diff-47ed1acddaad94538b9ee7995ffa8d7cc1376f9667350acee9cec912cec6a3bfR201
```

Comment:
Did you want to tackle this before merging?

Reply:
This one is for nixpkgs. Can't be done here.